### PR TITLE
Add ARMING_VOLT_MIN parameter to Power setup page

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

File: src/AutoPilotPlugins/APM/APMPowerComponent.qml [hunk 1]
```diff
@@ -29,6 +29,7 @@ SetupPage {
         Column {
             spacing: _margins
 
+            property Fact armVoltMin:       controller.getParameterFact(-1, "ARMING_VOLT_MIN")
```
Comment (jaxxzer) [inline src/AutoPilotPlugins/APM/APMPowerComponent.qml]: The name of this parameter changed: https://github.com/ArduPilot/ardupilot/commit/f5f6ec5
We need to handle that by looking for a parameter matching either name. Look at how the RC* and SERVO* parameters are handled.

Comment (dheideman) [reply src/AutoPilotPlugins/APM/APMPowerComponent.qml]: Reference: https://github.com/bluerobotics/qgroundcontrol/blob/master/src/FirmwarePlugin/APM/ArduSubFirmwarePlugin.cc

**Comment by dheideman** [comment]
Added input field for ARMING_VOLT2_MIN parameter, remapped old parameter names for older firmware versions.

**Comment by dheideman** [comment]
Removed ARMING_VOLT2_MIN input.

**Comment by jaxxzer** [comment]
Awesome. Can you please test this with a power module on a pixhawk running 3.5.1 to verify it works on the firmware side of things. Try with/without battery pre-arm check enabled. If it works, we can merge.

**Pull request merged**